Examples/ConnectionSample/AppDelegate.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 import Cocoa
-import SignalRClient
+import SwiftSignalRClient
```

Comment:
I was matching the name of the import as defined when using the library with Cocoapods. I changed the library module names so that it would be the same in all cases. More than happy to change that back and have SPM use the simplified 'SignalRClient' as well.

Reply:
I've updated all the references to just use 'SignalRClient'. The only place left that uses this name is the Cocoapods podspec, which already had that name defined.